tests/org.eclipse.passage.seal.demo.tests/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,16 @@
+Manifest-Version: 1.0
+Automatic-Module-Name: org.eclipse.passage.seal.demo.tests
+Bundle-ManifestVersion: 2
+Bundle-SymbolicName: org.eclipse.passage.seal.demo.tests
+Bundle-Version: 0.5.0.qualifier
+Bundle-Name: %Bundle-Name
+Bundle-Vendor: %Bundle-Vendor
+Bundle-Copyright: %Bundle-Copyright
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Require-Bundle: org.junit;bundle-version="4.12.0",
+ org.eclipse.osgi.services;bundle-version="0.0.0",
+ org.eclipse.passage.seal.demo;bundle-version="0.0.0",
+ org.eclipse.passage.lic.api.tests;bundle-version="0.0.0",
+ org.eclipse.passage.lic.api,
```

Comment:
I assume this requires API 1.0

Reply:
It only uses _internal_ parts not yet pushed to the official api level. Current `lic.api` is of version _0.7.0_. Can you please explain your expectation in some details?